app/Http/Controllers/QuizAPIController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Quiz;
+use Exception;
+use Illuminate\Http\JsonResponse;
+
+class QuizAPIController extends Controller
+{
+    public function displayAllQuizzes():JsonResponse
+    {
+        try {
+        $quizzes = Quiz::all();
+        } catch(Exception $e) {
+            return response()->json([
+                "message" => "Database not found"
```

Comment:
You shouldn't be this specific. It doesn't help the user - they can't do anything about your database. There are also other reasons your code could throw. Give external observers as little information as you can about the internal state of your server.